src/commands/watch/uppificator.rs

```diff
@@ -0,0 +1,110 @@
+use std::path::PathBuf;
+use uuid::Uuid;
+
+pub(crate) struct Uppificator {
+    pub spin_bin: PathBuf,
+    pub up_args: Vec<String>,
+    pub manifest: PathBuf,
+    pub clear_screen: bool,
+    pub watched_changes: tokio::sync::watch::Receiver<Uuid>,
+    pub pause_feed: tokio::sync::mpsc::Receiver<Pause>,
+    pub stopper: tokio::sync::watch::Receiver<Uuid>,
+}
+
+#[derive(Debug)]
+pub(crate) enum Pause {
+    Pause,
+    Unpause,
+}
+
+enum UppificatorAction {
```

Comment:
Uh... I don't know... I tend to do it when I need it.  Though there's no harm in doing it pre-emptively in case someone else - some kind, generous, forgiving Canadian for example - later needs to debug one's awful code and would prefer not to have to whack the derive on themself.